### DISCUSSION TITLE
[Feature request] Custom homebox-wide fields?

### DISCUSSION THREAD

**Discussion by YoYoZ** [sysadminsmedia/homebox#274] [Ideas]
Name speaks for itself. If possible, I'd love to see an ability to add new Custom fields or, at least, rename existing.

**Comment by tankerkiller125** [comment]
Custom fields are a feature once you've created an item. (Edit, way down at the bottom)

Once thing I would like to see is an item template of some kind so that you can create templates for specific kinds of items and fill those fields out during creation.

**Reply by PathosEthosLogos** [reply]
> Custom fields are a feature once you've created an item. (Edit, way down at the bottom)

Sorry, where is this? When I create a custom field, it's only for one item.

Either way is great though. It would be really nice if this gets implemented.

**Reply by dog-rgb** [reply]
I think the workflow with item templates is already available (see https://homebox.software/en/changelog/version/v0-21-0...latest/#item-templates )

But I would like to see the possibility to select from a list of custom fields which are used on other items already. With this I am able to avoid typos when adding custom fields later to an existing item.
